digits/scheduler.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, gpu_list=None, verbose=False):
         gpu_list -- a comma-separated string which is a list of GPU id's
         verbose -- if True, print more errors
         """
-        self.jobs = []
+        self.jobs = dict()
```

Comment:
I think we're going to want an `OrderedDict` here. If we add them in order when loading from `jobs_dir` then we can avoid calls to `sort()` elsewhere.